eva_submission/xlsx/xlsx_parser_eva.py

```diff
@@ -74,7 +74,8 @@ def references(self):
     def samples_per_analysis(self):
         samples_per_analysis = defaultdict(list)
         for row in self.samples:
-            samples_per_analysis[row.get('Analysis Alias')].append(row)
+            for analysis_alias in row.get('Analysis Alias').split(','):
+                samples_per_analysis[analysis_alias].append(row)
```

Comment:
This is the actual functional change

Reply:
Suggest trimming whitespace before and after commas to guard against whitespace junkies like me :)

Reply:
Done.